public/js/components/FormFields/__snapshots__/FormFieldImageSelect.spec.js.snap

```diff
@@ -58,11 +58,17 @@ exports[`test Should render with image when provided 1`] = `
           1215
         </div>
         <button
-          className="image-select__button"
+          className="image-select__button image-select__replace-button"
           onClick={[Function]}
           type="button">
           Replace Image
         </button>
+        <button
+          className="image-select__remove-button btn btn--red"
```

Comment:
dumb question - why does remove have a `btn` class but replace doesn't? is it needed?